JavaScript/chapter01/p01_is_unique/ada11.js

```diff
@@ -0,0 +1,30 @@
+
+
+ /* Write a function isUnique that:
+Input: takes an array of integers
+Output: returns a deduped array of integers
+*/
+
+ // using an Object
+const isUnique = (arr)=>{
+ let obj = {}
+ for (let elem of arr){
+   if(obj.hasOwnProperty(elem)) obj[elem]++
+   else obj[elem] = 1
+ }
+
+ return Object.keys(obj)
+ }
+
+//test cases
+const assert = require('assert');
+  describe(module.filename, () => {
+    it("should return true on an input string with unique characters", () => {
+      assert.equal(isUnique1("tech"), true);
+      assert.equal(isUnique2("tech"), true);
```

Comment:
```suggestion
      assert.equal(isUnique("tech"), true);
```

there's only one implementation in this file. same below.